src/lib/mongoBackend/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ SET (SOURCES
     compoundResponses.cpp
     location.cpp
     compoundValueBson.cpp
+    dateExpiration.cpp
```

Comment:
This new module in mongoBackend should be added to https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/mongoBackend.md#specific-purpose-modules. Maybe copy-pasting the same as `location` will be enougth, as the module seems to be quite similar.

In addition, it would be great if you could have a general look to the flows in https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/devel/mongoBackend.md in order to see if some one would need adaptation. I don't think so, as TTL management index seems to be a secondary aspect and the flows doen't typically go into so deeper detail. But please check anyway  (hint: look for usages of the location.cpp/h module in the flows).

Reply:
Fixed in ae70f62